internal/ignition/default.go

```diff
@@ -54,11 +55,16 @@ systemd:
         WantedBy=multi-user.target
 storage:
   files: []
-passwd: {}
+passwd:
+  users:
+    - name: metal
+      password_hash: "$6$nq7WpLYJhmsUn7Pa$SDFlU/34qMoEnscBK03Fqc26/RaV01QYB2dMvIgoRTFmNNYQVg7YYzTOG4phKAgMw5UwRU4ePwKndYea9X/iO0"
```

Comment:
Is a password required?

Reply:
Unfortunatelly yes. But I am not also generating a unique bycrypted passwd hash for each `Server`.